src/cirq_iqm/devices/iqm_device.py

```diff
@@ -92,7 +92,7 @@ def check_qubit_connectivity(cls, operation: cirq.Operation) -> None:
         """Raises a ValueError if operation acts on qubits that are not connected.
         """
         if len(operation.qubits) >= 2 and not isinstance(operation.gate, ops.MeasurementGate):
-            connection = set(cls.get_qubit_index(q) for q in operation.qubits)
+            connection = set(cls.get_qubit_index(q) for q in operation.qubits)  # type: ignore
```

Comment:
Why not change the interface of `get_qubit_index`? This is the only place where it is used.